src/api-utils.js

```diff
@@ -2,8 +2,40 @@ import request from 'superagent';
 
 const URL = 'https://tranquil-mountain-58489.herokuapp.com';
 
+export async function getCategoriesById(category_id) {
+    const categories = await request.get(`${URL}/categories`);
+    
+    const categoryName = categories.body.find(category => category.id === category_id);
+    return categoryName;
+}
+
 export async function getCocktails() {
     const response = await request.get(`${URL}/cocktails`);
 
     return response.body
+}
+
+export async function getCocktailById(id) {
+    const response = await request.get(`${URL}/cocktails/${id}`);
+
+    return response.body;
+}
+
+export async function createCocktail(newCocktail) {
+    const response = await request.post(`${URL}/cocktails`)
+    .send(newCocktail);
+
+    return response.body;
+}
+
+export async function updateCocktail(id) {
```

Comment:
Here we also want to pass the `updatedCocktail` so we can `.send(updatedCocktail)` it with the request :)